playground/passwordGen.cpp

```diff
@@ -8,31 +8,31 @@
 
 using namespace std;
 
-class passwords
+class Password
```

Comment:
Classes in literally every programming language are declared using PascalCase and in singular.

https://www.elpassion.com/blog/naming-101-programmers-guide-on-how-to-name-things